lib/ReactTags.js

```diff
@@ -251,18 +251,15 @@ class ReactTags extends Component {
     }
   }
 
-  handlePaste(e) {
-    if (!this.props.allowAdditionFromPaste) {
-      return;
-    }
-
-    e.preventDefault();
-
+  /**
+   * Used to determine how the pasted content is split.
+   * @returns {RegExp} Regular expression
+   */
+  get delimiterRegex() {
```

Comment:
any reason for using getter here ?
can we move this to a new file `utils.js` and rename this to `getDelimiterRegex(delimters)` ?

Reply:
it prevents the regex from being re-generated every time someone goes to paste. happy to move it to a utils, but I think it should remain a getter in this class

Reply:
Hmm I get your point. But as its not an expensive operation which if regenerated will cause issue when it goes to paste and also this looks like something which can be reused  in future so IMO it can be moved to utils. Also if the value of props change in the parent component then the value has to be recalculated so I don't think we will get any benefits of using getter here as we are not memoizing it here. 
Also since you are not memoizing the result so the getter will be executed every time the handlePaste executes. Does this make sense  @jpbufe3 ?

Reply:
awesome. So you can directly import `buildRegexForDelimiters` from utils and directly use `buildRegexForDelimiters` in handlePaste.